lib/config.js

```diff
@@ -1,9 +1,14 @@
+var service_host = process.env.MONGO_SERVICE_HOST
+
 var database = {
-    url: 'mongodb://localhost:27017/pacman',
+    url: `mongodb://${service_host}:27017/pacman`,
     options: {
-        replicaSet: 'rs0',
         readPreference: 'secondaryPreferred'
     }
 };
 
+if(process.env.MONGO_REPLICA_SET) {
+    database.options.replicaSet = process.env.MONGO_REPLICA_SET
```

Comment:
Only configure the replicaSet option when the MONGO_REPLICA_SET env var is set to enable working with the single cluster example again.